rotkehlchen/chain/evm/decoding/gearbox/gearbox_cache.py

```diff
@@ -277,8 +296,21 @@ def query_gearbox_data_from_chain(
     pools_data = evm_inquirer.contracts.contract(
         CHAIN_ID_TO_DATA_COMPRESSOR[evm_inquirer.chain_id],
     ).call(node_inquirer=evm_inquirer, method_name='getPoolsV3List')
-    new_pools = []
+    new_pools: list[GearboxPoolData] = []
+    last_notified_ts = 0
     for pool_data in pools_data:
+        if (current_time := ts_now()) - last_notified_ts > 5:
+            msg_aggregator.add_message(
+                message_type=WSMessageType.PROTOCOL_CACHE_UPDATES,
+                data={
+                    'protocol': CPT_GEARBOX,
+                    'chain': evm_inquirer.chain_id.to_name(),
```

Comment:
don't call .to_name on each loop iteration. Calling functions in python has a cost, define it as a variable before the loop